From 01bbcc359cfee7c38514f04182a1b33f79adb983 Mon Sep 17 00:00:00 2001
From: Lei Xu <lei@apache.org>
Date: Fri, 26 Feb 2016 10:47:49 -0800
Subject: [PATCH 1471/2748] HADOOP-12841. Update s3-related properties in
 core-default.xml. (Wei-Chiu Chuang via lei)

(cherry picked from commit 2093acf6b659d5a271b7e97f9b64652d7cf01eef)

Change-Id: I1ddec7bb623915525f23e256ee2ddec7de390fac
---
 .../src/main/resources/core-default.xml            |   28 +++++++++++++++++---
 1 file changed, 24 insertions(+), 4 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/resources/core-default.xml b/hadoop-common-project/hadoop-common/src/main/resources/core-default.xml
index 75e197c..e377dc8 100644
--- a/hadoop-common-project/hadoop-common/src/main/resources/core-default.xml
+++ b/hadoop-common-project/hadoop-common/src/main/resources/core-default.xml
@@ -636,6 +636,16 @@ for ldap providers in the same way as above does.
 </property>
 
 <property>
+  <name>fs.s3.awsAccessKeyId</name>
+  <description>AWS access key ID used by S3 block file system.</description>
+</property>
+
+<property>
+  <name>fs.s3.awsSecretAccessKey</name>
+  <description>AWS secret key used by S3 block file system.</description>
+</property>
+
+<property>
   <name>fs.s3.block.size</name>
   <value>67108864</value>
   <description>Block size to use when writing files to S3.</description>
@@ -682,6 +692,16 @@ for ldap providers in the same way as above does.
 </property>
 
 <property>
+  <name>fs.s3n.awsAccessKeyId</name>
+  <description>AWS access key ID used by S3 native file system.</description>
+</property>
+
+<property>
+  <name>fs.s3n.awsSecretAccessKey</name>
+  <description>AWS secret key used by S3 native file system.</description>
+</property>
+
+<property>
   <name>fs.s3n.block.size</name>
   <value>67108864</value>
   <description>Block size to use when reading files using the native S3
@@ -722,13 +742,13 @@ for ldap providers in the same way as above does.
 </property>
 
 <property>
-  <name>fs.s3a.awsAccessKeyId</name>
-  <description>AWS access key ID. Omit for Role-based authentication.</description>
+  <name>fs.s3a.access.key</name>
+  <description>AWS access key ID used by S3A file system. Omit for Role-based authentication.</description>
 </property>
 
 <property>
-  <name>fs.s3a.awsSecretAccessKey</name>
-  <description>AWS secret key. Omit for Role-based authentication.</description>
+  <name>fs.s3a.secret.key</name>
+  <description>AWS secret key used by S3A file system. Omit for Role-based authentication.</description>
 </property>
 
 <property>
-- 
1.7.9.5

